presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/RemoveRedundantAggregateDistinct.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Similarly all these Remove* rules seem to be a like a group of rules that can be applied together. Would be nice to may be write a separate rulegroup that has all these in one place for logically grouping rules.

Reply:
Would it be ok if we did that in a separate PR? Please open an issue. Thanks.